modules/apps/info/info-api/src/main/java/com/liferay/info/item/InfoItemFormVariation.java

```diff
@@ -32,7 +31,7 @@ public InfoItemFormVariation(
 
 		this(key, labelInfoLocalizedValue);
 
-		_groupIdOptional = Optional.of(groupId);
+		_groupId = groupId;
 	}
 
 	public InfoItemFormVariation(
```

Comment:
@JanisZhang Please investigate whether you can remove this constructor. If so, remove it, so that you can make `_groupId` of type `long` instead of `Long`. With this, you don't need to do null check when getting groupId from this object.

Reply:
Hi @dantewang, Sorry I don't get this -> "`remove it, so that you can make _groupId of type long instead of Long`". 
Is there any relationship between that? Thank you